src/auto/accel_group.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (b7f5189) from gir-files (71d73f0)
+// This file was generated by gir (b7f5189) from gir-files (71d73f0+)
```

Comment:
Hm, I missed that + in gir-files version.
Check `git status` in `gir-files` directory: it must say `nothing to commit, working tree clean`
After this remove `gir/target` and try rebuild/